src/handleSubmit.js

```diff
@@ -32,6 +35,9 @@ const handleSubmit = (submit, props, valid, asyncValidate, fields) => {
             return submitResult
           }, submitError => {
             const error = submitError instanceof SubmissionError ? submitError.errors : undefined
+            if (!error) {
+              setTimeout(() => throw submitError)
+            }
             stopSubmit(error)
```

Comment:
The `setTimeout()` solution is slightly opinionated. With this solution, if some kind of unexpected error like `TypeError`, `ReferenceError` and other system errors, this will actually give you file name and line number of the original error. If you just do a regular throw/catch through the promise you lose that information. I'm happy to conform to whatever standard the team feels is best.